corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -127,7 +116,7 @@ def _datadog_timing(self, step):
             timing_buckets=(.03, .1, .3, 1, 3, 10),
             tags={
                 'action': step,
-                'index': self.index_info.alias,
+                'index': self.adapter.index_name,  # TODO: try to add index cname
```

Comment:
> TODO: try to add index cname

using the actual index name is probably more useful since you could tell if a new index is slower than an old index